simulation/core/create_index.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2023 MosaicML Streaming authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""Create a dataset index file from input parameters."""
+
+import os.path
+import sys
+
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
```

Comment:
Why do we need this?

Reply:
Oh yeah I put this here by mistake. Within the `core` directory this isn't needed, but when we want to import functionality from the `core` directory in the `interfaces` or `testing` directories then it's necessary. Otherwise we get import errors.